client/src/components/Toasts/Toast.test.js

```diff
@@ -67,23 +70,69 @@ describe('Notification Toast', () => {
       })
     );
     expect(queryByRole('alert')).toBeDefined();
-    expect(queryByRole('alert')).toHaveTextContent(/RStudio-S...cvserver is now running/);
-    expect(queryByRole('alert')).not.toHaveTextContent(/RStudio-S...cvserver finished successfully/);
+    expect(queryByRole('alert')).toHaveTextContent(
+      /RStudio-S...cvserver is now running/
+    );
+    expect(queryByRole('alert')).not.toHaveTextContent(
+      /RStudio-S...cvserver finished successfully/
+    );
+  });
+});
+
+describe('Toast Message', () => {
+  it('shows data file error toast message', () => {
+    const { getByText } = renderComponent(
+      <ToastMessage notification={dataFilesError} />,
```

Comment:
Would it be possible to provide the dataFilesError object to the higher level NotificationToast component? I can see pros and cons of both, notable cons being:
- least concerns testing
- not being able to query the toast itself.

If these cons are too big then ignore this comment.

Reply:
Good idea.  Moved all the added tests to the NotificationToast testing.